NachoSpigot-Server/src/main/java/net/minecraft/server/EntityArrow.java

```diff
@@ -452,7 +452,7 @@ public void d(EntityHuman entityhuman) {
             }
 
             if (flag) {
-                world.makeSound(this.shooter,"random.pop", 0.2F, ((this.random.nextFloat() - this.random.nextFloat()) * 0.7F + 1.0F) * 2.0F);
+                world.makeSound(entityhuman,"random.pop", 0.2F, ((this.random.nextFloat() - this.random.nextFloat()) * 0.7F + 1.0F) * 2.0F);
```

Comment:
We should probably just change it to default vanilla behavior. (everyone nearby hears it)
```suggestion
                this.makeSound("random.pop", 0.2F, ((this.random.nextFloat() - this.random.nextFloat()) * 0.7F + 1.0F) * 2.0F);
```